frontend/src/Attribution/Plots.elm

```diff
@@ -146,12 +153,31 @@ plotNgramsSpi { ngramsSpi } =
             |> viewBars (groups (List.map (uncurry group)))
 
 
+plotSimilarities : Statistics -> Html.Html msg
+plotSimilarities { similarity } =
+    let
+        rename : String -> String
+        rename str =
+            str
+                |> Regex.replace Regex.All (Regex.regex "\"") (\_ -> "")
+                |> Regex.replace Regex.All (Regex.regex "_") (\_ -> " ")
+
+        construct ( key, value ) =
+            ( rename (toString key), [ value ] )
```

Comment:
if you remove the call to toString, the first regex line is not needed, because there will be no \" in the string